fundamentals/arrays/lesson_notes/lesson_01.md

```diff
@@ -0,0 +1,137 @@
+# Arrays, Standard Operations
+
+## Setup
+
+Please complete the following before the lesson begins:
+
+- Fork this gist. Then, if you wish to take notes in the gist itself, click the “Edit” button.
+- Open and then fork this [Replit](https://replit.com/@bwreid/Arrays-Part-I).
+- Complete the Warmup Questions & Tasks detailed at the top of the Replit.
+
+## Learning Objectives
+
+By the end of this lesson you will be able to:
+
+- Describe the purpose of arrays and their unique properties.
+- Create new array literals with open and closing brackets.
+- Access elements of an array at a given index.
+- Access the length of an array.
+- Use array methods to remove elements from the front and back of an array.
+- Use array methods to add elements to the front and back of an array.
+- Assign values in an array at a particular index with bracket notation.
+
+## Guiding Questions
+
+- What can arrays do that strings, numbers, booleans, undefined, and null cannot do?
```

Comment:
I would either give the students all the comments/all the starter code or none of it and let them copy it all.

I worry that with the back and forth it might be easy to get lost/overwrite things by accident

Reply:
I'm unclear as to what you're saying here. Do you mean for the Warmup Questions?

Reply:
Yes, for the `repl.it` - there are notes to run code from the repl.it but also, I think I saw some that were not in the repl, but students were supposed to hypothesize and then run it? I guess just yes, more clarity on the flow for students - we can always hop on zoom to talk through if this is still confusing!